client/src/pages/Post/Post.tsx

```diff
@@ -42,13 +69,37 @@ export const Post = () => {
     );
   }
 
+  const handleChangePost = async (updatedPost: CreatedPost) => {
+    await editPost(post._id, updatedPost);
+    return setEditMode(false);
```

Comment:
it will be better if you'll separate those
```
await editPost(post._id, updatedPost);
setEditMode(false);
return;
```

It's like you know this funny example of javascript
`[1,2,3].map(parseInt) => [1, NaN, NaN]`
it happens because map callback has 3 arguments coming
`callback( currentValue[, index[, array]])`
and `parseInt` has 2 arguments `parseInt(string, radix);`
it's just because usually we don't use `radix` from `parseInt` and `index, array` from `map`, that's why the example looks unexpected and weird,

something similar can happen with `handleChangePost` method, you'll forget it returns something, React will change its API for useState callbacks, or you'll refactor something.
Never return anything if you don't actually need it because you can get weird behavior and then spend a lot of time debugging the bug.

Also when I read `return functionSmth();`, I try to realize what has been returned and why, when I see `return;` I skip it and understand that you just need to go out of function. It looks cleaner and more readable.

Reply:
Thanks a lot for your suggestions :)